CODEOWNERS

```diff
@@ -1,4 +1,4 @@
-* @Noxsios
+/* @defenseunicorns/uds-edge
 
 /CODEOWNERS @jeff-mccoy @daveworth
-/LICENSE @jeff-mccoy @daveworth
+/LICENS* @jeff-mccoy @austenbryan
```

Comment:
This license configuration assumes the dual licensing model as in here: https://github.com/defenseunicorns/uds-core/blob/main/LICENSING.md

So we should either pull in a `LICENSING.md` like other UDS Packages are revert this file-match (but changing to Austen is correct)